backend/src/utils/index.ts

```diff
@@ -1,19 +1,37 @@
-import express from 'express';
+import express, { NextFunction } from 'express';
 import { ErrorHandler } from '../services/error';
-import { Fields } from 'formidable';
-
-export const extractPayload = async (payload: express.Request): Promise<never | Fields | undefined> => {
+import { User } from './interfaces';
+export const extractPayload = async (payload: express.Request): Promise<never | Record<string, any> | undefined> => {
   try {
-    const isPayload = payload.fields !== undefined;
+    const isPayload = payload.body !== undefined;
     if (isPayload) {
-      return payload.fields as Fields;
+      return payload.body as Record<string, any>;
     } else {
       throw new ErrorHandler(400, { status: 'No Payload' });
     }
   } catch (e) {
     throw new ErrorHandler(400, { status: 'No Payload' });
   }
 };
+
+export const userExists = async (user: User): Promise<boolean> => {
+  return true;
```

Comment:
Is something missing here?

Reply:
Might not understand the syntax completely, but to me, it looks like the method simply returns true no matter what. Or is there a condition that the Promise must be upheld, or else an exception is thrown?

Reply:
It is a dummy function for now, it is not in use. i'll remove it, but i believe i'll need it later down the line.